src/scripts/main.js

```diff
@@ -1 +1,9 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
```

Comment:
Add a comment above the event listener to explain its purpose, e.g., // Manage menu visibility based on the URL hash. 📝